src/crypto/x509/pkix/pkix.go

```diff
@@ -260,6 +270,10 @@ func (n Name) String() string {
 					continue
 				}
 			}
+			if oidEqual(t, oidEmailAddress) {
```

Comment:
There is already an implementation for this:
```suggestion
			if t.Equal(asn1.ObjectIdentifier(oidEmailAddress)) {
```

Reply:
Thanks!